_build-and-test-all.sh

```diff
@@ -27,4 +27,27 @@ ${dockerall}Up
 
 ./gradlew :end-to-end-tests:cleanTest :end-to-end-tests:test
 
+./wait-for-services.sh localhost readers/${READER}/finished "8099"
+
+migration_file="migration_scripts/${DATABASE}/migration.sql"
+
+rm -f $migration_file
```

Comment:
A couple more comments.

* Lines 23-45 should be a script in eventuate-foundation/eventuate-common. No point in copy/pasting into multiple projects.
* Also, the output of curl can be piped into the cli.sh - no need to download a file.